src/compute/src/compute_state.rs

```diff
@@ -300,7 +300,7 @@ pub(crate) struct ActiveComputeState<'a, A: Allocate> {
 }
 
 /// A token that keeps a sink alive.
-pub struct SinkToken(Box<dyn Any>);
+pub struct SinkToken(#[allow(dead_code)] Box<dyn Any>);
```

Comment:
```suggestion
pub struct SinkToken {
  _guard: Box<dyn Any>,
}
```
This and the other sites triggering "dead code" warning are because the inner fields aren't _technically_ read, we just rely on their `Drop` impls getting called. If you make this a named field it should remove the need for `#[allow(dead_code)]`

Reply:
Yeah that'll be cleaner! Discussed offline with @ParkMyCar -- will address these in a follow up PR to get this update in asap since it'll take some time to update all instantiations of the relevant structs